src/test/java/com/odysseusinc/arachne/portal/service/analysis/heracles/parts/AnalysesQueriesRendererTest.java

```diff
@@ -0,0 +1,49 @@
+package com.odysseusinc.arachne.portal.service.analysis.heracles.parts;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.odysseusinc.arachne.portal.service.analysis.heracles.HeraclesAnalysisKind;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static com.odysseusinc.arachne.portal.service.analysis.heracles.parts.HeraclesTestUtils.prefixWithAt;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.when;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class AnalysesQueriesRendererTest {
+
+    private HeraclesAnalysesLoader.HeraclesAnalysis analysis111;
+    private HeraclesAnalysesLoader.HeraclesAnalysisParameter analysis111param;
+    @Mock
+    private HeraclesAnalysesLoader heraclesAnalysesLoader;
+    @InjectMocks
+    private AnalysesQueriesRenderer analysesQueriesRenderer;
+
+    @Before
+    public void setUp() {
+
+        analysis111 = new HeraclesAnalysesLoader.HeraclesAnalysis(111, "Number of persons by observation period start month", "111_112_numberOfPersosnByObservationPeriod.sql", true, false);
+        analysis111param = new HeraclesAnalysesLoader.HeraclesAnalysisParameter(111, "fieldName", "observation_period_start_date");
+    }
+
+    @Test
```

Comment:
this method name is not descriptive:
maybe something like  `shouldRenderValidAnalysis` and extract  111 as constant

Reply:
done